winml/api/Microsoft.AI.MachineLearning.Experimental.idl

```diff
@@ -3,15 +3,16 @@
 
 import "Windows.Foundation.idl";
 import "dualapipartitionattribute.idl";
-import "Windows.AI.MachineLearning.idl";
 
 #include <sdkddkver.h>
 
 #ifdef BUILD_INBOX
+import "Windows.AI.MachineLearning.idl";
 #define ROOT_NS Windows
 #define INBOX_ONLY(x) x
 #define OTB_ONLY(x)
 #else
+import "Microsoft.AI.MachineLearning.idl";
```

Comment:
I wonder how redist builds were working before without this change

Reply:
It was working because it was using Windows.AI.MachineLearning.idl which was getting pulled from the local folder.

Now the cppwinrt target function configures callers to use the winml_api build output location, and so it can use Microsoft.AI.MachineLEarning.idl (which was already being copied to the winml_api build output location).